--- origsrc/ghostscript-8.63/src/configure.ac	2008-08-17 23:19:38.354450000 -0500
+++ src/ghostscript-8.63/src/configure.ac	2008-08-18 01:00:29.433663900 -0500
@@ -82,7 +82,7 @@
 dnl --------------------------------------------------  
 
 AC_ARG_ENABLE(contrib, [  --disable-contrib       do not include contributed drivers [default=include]])
-CONTRIBINCLUDE="include contrib/contrib.mak"
+CONTRIBINCLUDE="include \$(CONTRIBDIR)/contrib.mak"
 INSTALL_CONTRIB="install-contrib-extras"
 if test x$enable_contrib = xno; then
     CONTRIBINCLUDE=""
@@ -445,7 +445,7 @@
     IJSDEVS=''
 if test x$with_ijs != xno; then
     AC_MSG_CHECKING([for local ijs library source])
-    if test -d ijs; then
+    if test -d $srcdir/ijs; then
         AC_MSG_RESULT([yes])
         IJSDIR=ijs
         IJSDEVS='$(DD)ijs.dev'
@@ -464,7 +464,7 @@
 if test x$with_jbig2dec != xno; then
   AC_MSG_CHECKING([for local jbig2dec library source])
   for d in jbig2dec jbig2dec-0.2 jbig2dec-0.3; do
-    test -d "$d" && JBIG2DIR=$d && break
+    test -d "$srcdir/$d" && JBIG2DIR=$d && break
   done
   if test "x$JBIG2DIR" != xsrc; then
     AC_MSG_RESULT([$JBIG2DIR])
